### DISCUSSION TITLE
AGW installation failed - code-killed, status=15/TERM

### DISCUSSION THREAD

**Discussion by vinothinfinity** [magma/magma#11988] [Q&A] [answered]
Hi folks,

 I am new to magma and trying to install AGW on ubuntu-20.04.4-live-server-amd64 (VBox over Windows 11)
 I followed the below steps,
 https://docs.magmacore.org/docs/lte/deploy_install#2-deploy-magma-on-the--agw_host

After Ubuntu server reloaded, i see below failure. Can someone give any pointers? not sure of the issue leading to failure.

![Screenshot 2022-03-07 064302](https://user-images.githubusercontent.com/23690519/156951714-81c6b941-5fd0-4e15-8023-d230256550c2.png)

Thanks in advance
Vinoth

**Comment by vinothinfinity** [comment, marked as answer]
Hi,

I tried below things and it worked.

1) create account name and password with "magma"
2) Use 2 CPUs for Ubuntu in VBox. More CPUs results in context switch.
3) Install VBox guess additions.

Thank You !!

**Reply by jni2000** [reply]
@vinothinfinity I encountered the same problem installing on Ubuntu 20.4. The difference is that I do not use vbox. My error screen is exactly the same as yours. Additionally,
1. the installation did create a "magma" account, but password unknown
2. after installation, my eth0 is not be able to bring up
Any clue ?
Thanks.
